### PR TITLE
feat: added Palestine location data 

### PR DESCRIPTION
**Pull request by charan-003**
feat: added Palestine location data 
* added Palestine location data

* added test case

### CODE REVIEW

**Review by cieslarmichal — APPROVED** [review]

**Comment by cieslarmichal** [comment]
please fix build

File: tests/modules/location_test.cpp [hunk 1]
```diff
@@ -82,6 +82,8 @@ CountryAddressesInfo getAddresses(const Locale& locale)
         return israelAddresses;
     case Locale::es_MX:
         return mexicoAddresses;
+    case Locale::Ar_PS;
```
Comment (cieslarmichal) [inline tests/modules/location_test.cpp:85]: colon instead of semicolon

**Review by cieslarmichal — CHANGES_REQUESTED** [review]

**Review by charan-003 — COMMENTED** [review]
done

**Review by charan-003 — COMMENTED** [review]
changed the variable from Ar_PS to ar_PS, from palestineAddresses to PalestineAddresses

File: src/modules/location_data.h [hunk 2]
```diff
@@ -8234,6 +8234,185 @@ const CountryAddressesInfo mexicoAddresses{
     {},
     (mexicoStates),
 };
+// Palestine
+const auto PalestineCities = std::to_array<std::string_view>(
```
Comment (cieslarmichal) [inline src/modules/location_data.h:8238]: please use camelCase with first character lower as variable names

**Review by charan-003 — COMMENTED** [review]
fixed all the camelCasing isseus

**Review by charan-003 — COMMENTED** [review]
.

**Comment by charan-003** [comment]
hi,
I hope this works now

**Review by charan-003 — COMMENTED** [review]
changed the postal code format

**Review by cieslarmichal — APPROVED** [review]

**Pull request merged**